torch/distributions/binomial.py

```diff
@@ -0,0 +1,94 @@
+from numbers import Number
+import torch
+import math
+from torch.distributions import constraints
+from torch.distributions.distribution import Distribution
+from torch.distributions.utils import broadcast_all, probs_to_logits, lazy_property
+from torch.distributions.utils import clamp_probs
+from torch.autograd import Variable
+
+
+class Binomial(Distribution):
+    r"""
+    Creates a Binomial distribution parameterized by `total_count` and
+    either `probs` or `logits` (but not both).
+
+    -   :meth:`sample` requires a single shared `total_count` for all
+        parameters and samples.
+
+    Example::
+
+        >>> m = Binomial(100, torch.Tensor([0 , .2, .8, 1]))
+        >>> x = m.sample()

```

Comment:
Could you update these with a plausible sample? it should look like `[0, 22, 74, 100]`.